src/accessories/contact-sensor.js

```diff
@@ -0,0 +1,35 @@
+import 'babel-polyfill';
+import {HomeWizardBaseAccessory} from './accessory';
+
+export class HomeWizardContactSensor extends HomeWizardBaseAccessory {
+
+  model = 'Contact sensor';
+
+  setupServices() {
+    // Setup services
+    const contactSensorService = new this.hap.Service.ContactSensor();
+    contactSensorService
+      .getCharacteristic(this.hap.Characteristic.ContactSensorState)
+      .on('get', this.getContactSensorState.bind(this));
+
+    this.services.push(contactSensorService);
+  }
+
+  getContactSensorState(callback) {
+    this.api.getStatus(this.id, 'kakusensors').then(sensor => {
+      const contact = sensor.status === null
+        ? this.hap.Characteristic.ContactSensorState.CONTACT_NOT_DETECTED
+        : this.hap.Characteristic.ContactSensorState.CONTACT_DETECTED;
+
+      if (contact === this.hap.Characteristic.ContactSensorState.CONTACT_DETECTED) {
+        this.log(`Detected contact at sensor:${this.name}`);
```

Comment:
I will change the code to log al the time because it depends of the usage. 
You can use it like a door sensor when the door is usually closed.
Or the inverse when the contact is rare.
For me the log is especially a debugging tool, not useful in the true life !

Another solution for ALL the sensors is to memorize in the object the status and log only when it change.